corehq/apps/es/tests/utils.py

```diff
@@ -51,7 +57,8 @@ class ElasticTestMixin(object):
     def setUpClass(cls):
         super().setUpClass()
         cls._es_instance = get_es_new()
-        initialize_index_and_mapping(cls._es_instance, TEST_INDEX_INFO)
+        with ignore_index_settings_key_warning:
```

Comment:
This will eventually be removed, right? Maybe add a comment here to note that?

Reply:
See c585b24d403